stl/inc/system_error

```diff
@@ -554,9 +553,33 @@ _NODISCARD const _Ty& _Immortalize_memcpy_image() noexcept {
 #elif defined(__cpp_constexpr_dynamic_alloc)
 template <class _Ty>
 _NODISCARD const _Ty& _Immortalize_memcpy_image() noexcept {
-    static constexpr _Ty _Static;
+#ifdef __cpp_constinit
+    constinit
+#endif // __cpp_constinit
+        static constexpr _Ty _Static;

```

Comment:
The tests are failing with:
```
D:\build\x86\out\inc\system_error(559,16): error: cannot combine with previous 'constinit' declaration specifier
        static constexpr _Ty _Static;
               ^
```

Reply:
[[dcl.spec]/2](http://eel.is/c++draft/dcl.spec#2): "At most one of the `constexpr`, `consteval`, and `constinit` keywords shall appear in a _decl-specifier-seq_."

Reply:
That's the last time I blindly apply @CaseyCarter's suggestion I guess :P

Reply:
Here's a new suggestion (feel free to apply at full visibility):
```suggestion
    static constexpr _Ty _Static;
```